src/lib/ursadb.py

```diff
@@ -15,11 +19,15 @@ def make_socket(self, recv_timeout=2000):
         socket.connect(self.backend)
         return socket
 
-    def query(self, query):
+    def query(self, query, taint):
         socket = self.make_socket(recv_timeout=-1)
 
         start = time.clock()
-        query = "select {};".format(query)
+        if taint:
+            taint = taint.replace('"', '"')
```

Comment:
Is there a situation in which this isn't a NOP? If that's the case, perhaps a small comment explaining it?

Reply:
:thinking: what the... Yeah, that is supposed to be: 

```python
taint = taint.replace('"', '\\"')
```